lib/scss/components/news-list-item/_list-item.scss

```diff
@@ -131,7 +214,7 @@
   }
 }
 
-.su-news-list__footer {
+.su-news-edit-article {
```

Comment:
Where does this selector come from?

Reply:
It is in the component. We set it here in the pattern. 
![image](https://user-images.githubusercontent.com/1187335/120040678-dc417900-bfbb-11eb-8c0f-8d8d466b5920.png)

Reply:
Yeah this is another example of using a selector to target something that isn't in the pattern template. IMO the pattern's styles should not alter or target anything that isn't directly part of the pattern.